plugins/tracker-resources/src/index.ts

```diff
@@ -230,7 +230,20 @@ async function deleteProject (project: Project | undefined): Promise<void> {
         }
       )
     } else {
-      await removeProject(project)
+      showPopup(
```

Comment:
Why not use generic method to ask for confirmation and delete? Let's remove specialised one. This should be generic for any space. As well it should trigger delete of all child objects in space.